src/ui/pages/Settings/Settings.tsx

```diff
@@ -340,6 +343,44 @@ function ToggleSettingLine({
   );
 }
 
+function ClearPendingTransactionsLine() {
+  const toastRef = useRef<PopoverToastHandle>(null);
+  const { mutate: clearPendingTransactions, ...mutation } = useMutation({
+    mutationFn: async () => {
+      await new Promise((r) => setTimeout(r, 500)); // artificial delay
+      toastRef.current?.removeToast();
+      return walletPort.request('clearPendingTransactions');
+    },
+    onSuccess: () => {
+      toastRef.current?.showToast();
+    },
+  });
+  return (
+    <>
+      <PopoverToast ref={toastRef}>Pending transactions cleared</PopoverToast>
+
+      <Frame
+        as={UnstyledButton}
+        interactiveStyles={true}
```

Comment:
May be we can name this just `interactive`? Usually we use ___Styles for CSSProperties param

Reply:
I would leave the name as it is. Ideally, if `interactiveStyles` doesn't make sense without having `as={...Button}` or `as={...Link}` then it should be impossible to construct such a `Frame` component (I mean it shouldn't type-check in this case)